### ISSUE TITLE
Bug - sky map will glitch if no valid tile to travel ro was found

### ISSUE THREAD

**Issue by Doronimmo** [evanbowman/skyland-beta#111]
To recreate: go to a dead end without a radar and let the storm get the previous tile so there is no valid place to travel

**Comment by evanbowman** [comment]
Yeah I know about that bug, I've been meaning to fix it. It's mostly a graphics glitch that doesn't affect the game otherwise, so I've been lazy about fixing it
